### PR TITLE
Fix more ByteArray warnings

### PR DESCRIPTION
**Pull request by dylanmccall**

### CODE REVIEW

File: js/framework/application.js [hunk 1]
```diff
@@ -475,12 +475,14 @@ var Application = new Knowledge.Class({
...
+        let [, stdout_bytes, stderr_bytes, status] = GLib.spawn_command_line_sync(SCSS_COMMAND + theme_file.get_path());
         if (status !== 0) {
-            printerr(new Error(stderr.toString()));
+            let stderr = ByteArray.toString(stderr_bytes)
+            printerr(new Error(stderr));
             System.exit(1);
         }
-        return stdout.toString();
+        let stdout = ByteArray.toString(stdout_bytes);
+        return stdout;
```
Comment (dnohales) [inline js/framework/application.js]: I would just do `return ByteArray.toString(stdout_bytes);`, same in other places

File: tools/kactus [hunk 2]
```diff
@@ -96,11 +97,12 @@ function show_message (text) {
     dialog.destroy();
 }
 
-function read_sream (stream) {
+function read_stream (stream) {
```
Comment (dnohales) [inline tools/kactus]: :scream:

File: tools/kactus [hunk 3]
```diff
@@ -96,11 +97,12 @@ function show_message (text) {
...
     let output = '', buffer;
     do {
         buffer = stream.read_bytes(1024 * 8, null);
-        output += buffer.get_data().toString();
+        output_uint8array = buffer.get_data();
+        output += ByteArray.toString(output_uint8array);
```
Comment (dnohales) [inline tools/kactus]: `output += ByteArray.toString(buffer.get_data());` is more readable.

**Comment by dylanmccall** [comment]
Good points :) I changed those to be more straight-forward now.

**Pull request merged**